tgp-msg.c

```diff
@@ -193,7 +193,16 @@ static char *format_document (const char *path, const char *filename, const char
   gchar *pth = g_markup_escape_text (path, -1);
   gchar *fle = g_markup_escape_text (filename, -1);
   gchar *mme = g_markup_escape_text (mime, -1);
-  gchar *fsize = g_format_size (size);
+  gchar *fsize =
+#if GLIB_CHECK_VERSION(2,30,0)
+    g_format_size (size)
+#elif GLIB_CHECK_VERSION(2,16,0)
+    /* Necessary as we compile on windows with glibc 2.24.2 */
```

Comment:
> glib != glibc

Oh shit.  Now I need to go through it all again, but I think the comment is wrong here.
Thanks for pointing it out.

> Your version check is for 2.16 but the comment is 2.14?

The comment is for version 2.24, since it's the version at which … you know what, I shouldn't have to explain this.  If I have to justify it *here*, then my code sucks.

I'm ashamed of not having written a comment.  I'll try again soon.  Spoiler: there's a good reason for the version mismatch.